### DISCUSSION TITLE
(suggestion)projectile aimbot - feet aim on ground - wait for legs to be visible

### DISCUSSION THREAD

**Discussion by JDNA364PROC** [Fedoraware/Fedoraware#982] [Ideas]
it's annoying af when you have it on with direct hit it mostly owns walls and ground

it would be cool if with this on and legs wouldn't be visible it would focus on other part of the body or wait for legs to be visible.

your choice to do that i don't force

**Comment by BLS01** [comment]
😶

**Comment by ghost** [comment]
Good feature 👍
